### ISSUE TITLE
[FEATURE] Add ability to provide a custom context

### ISSUE THREAD

**Issue by japborst** [luunc/flutter-drag-to-pop#1]
In some cases you'd want to use a different context upon open in order to route the screen differently on tap.

Would be great if that can be added as a parameter.

**Comment by luunc** [comment]
@japborst , I'm not sure what you mean.
Can you me an example or demonstration for that?

**Comment by japborst** [comment]
Of course. Especially when you have nested routes, you would like to control the Navigator that the screen is routed from. Namely, you want to use the root navigator and not a nested one for an actual full screen experience.

In that case, you could do something like:

```dart
DragToPop(
  closeWidget: ...
  openWidget: ...,
  context: customContext
);
```

... such that the `Navigator` can use this. Something like

```dart
Widget build(BuildContext context) => Hero(
  tag: heroKey,
  child: GestureDetector(
    child: closeWidget,
    onTap: () => Navigator.of(this.context ?? context).push(...)
));
```

**Comment by luunc** [comment]
@japborst , Sorry for late response.
Currently, this package is not supported anymore.
Please consider using this one: [overscroll_pop](https://pub.dev/packages/overscroll_pop) which is mine also.
This package give you freedom to use without limit of context.
